src/doc/unstable-book/src/macro-vis-matcher.md

```diff
@@ -0,0 +1,14 @@
+# `macro_vis_matcher`
+
+The tracking issue for this feature is: [#41022]
+
+With this feature gate enabled, the [list of fragment specifiers][frags] gains one more entry:
+
+* `vis`: a visibility qualifier. Examples: nothing (default visibility); `pub`; `pub(crate)`.
+
+A `vis` variable may be followed by a comma, ident, type, or path.
```

Comment:
To quote @DanielKeep's rationale from above:

> I allowed comma because having some kind of sequence-delimiting token is useful. If you're passing bits of parsed input around, you might have `inner!($thing, $a_vis, $more_tts)`.

This isn't required, we can exclude it and you'd have to use `($a_vis)` to pass around parsed visibility fragment within macros.